docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -25,7 +25,29 @@ Refresh strategy can be customized by setting the
 pre-aggregation.
 
 The default value of `refreshKey` is `every: '1 hour'`. It can be redefined
-either by providing SQL:
+either by overriding the default value of the [`every`
+property][ref-schema-ref-preaggs-refresh-key-every]:
+
+```javascript
+cube(`Orders`, {
+  // ...
+
+  preAggregations: {
+    amountByCreated: {
+      type: `rollup`,
+      measures: [amount],
+      timeDimension: createdAt,
+      granularity: `month`,
+      refreshKey: {
+        every: `12 hour`,
+      },
+    },
+  },
+});
+```
+
+Or by providing a [`sql` property][ref-schema-ref-preaggs-refresh-key-sql]
```

Comment:
we need to confirm with core and then mention it here - 

**what is the default value of every when only `sql` is specified?**

I assume it is 1 hour, but worth to double check.

Reply:
I'll add a link to https://cube.dev/docs/caching#in-memory-cache-default-refresh-keys which defines the defaults